rpc/rpc-wrappers.go

```diff
@@ -372,6 +374,23 @@ func NewRegistrationAuthorityServer(rpc Server, impl core.RegistrationAuthority,
 		return
 	})
 
+	rpc.Handle(MethodDeactivateAuthorization, func(ctx context.Context, req []byte) (response []byte, err error) {
```

Comment:
I think using named returns for this one & matching the style in-file is the right approach. We should clear out all of the named returns in this file at once as another issue: https://github.com/letsencrypt/boulder/issues/2133